### DISCUSSION TITLE
Segfaults when getting started on Linux

### DISCUSSION THREAD

**Discussion by rsgrimes** [Embarcadero/DelphiFMX4Python#22] [General]
I wanted to try DelphiFMX4Python on my Ubuntu 18.04 LTS system but ran into problems immediately. After installing it using `pip install delphifmx`, I tried to run the simplest example and it segfaulted, immediately upon attempting to import the module; for example:

```$ python
Python 3.8.6 (default, Oct  6 2020, 03:22:36) 
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from delphifmx import *
Segmentation fault (core dumped)
```

@lmbelo asked about whether I had installed Python (which of course, I had), telling me that unfortunately, "the Python version distributed by apt-get is not supported." 

If you are running in to this problem, you may want to install a supported version of Python using these [scripts](https://github.com/Embarcadero/DelphiFMX4Python/tree/main/scripts), which are in the repo. (Note: I found a version of Python 3.7 on my system that worked, so I didn't try these scripts myself, but the scripts probably are an easy option that you can consider.)

**Comment by lmbelo** [comment]
Hi all,

we have a release candidate fixing this issue. Download it directly from test.pypi:

`pip install -i https://test.pypi.org/simple/ delphifmx==1.0.6`
